pkg/kubelet/dockertools/manager.go

```diff
@@ -1497,7 +1499,7 @@ func (dm *DockerManager) createPodInfraContainer(pod *api.Pod) (kubeletTypes.Doc
 		return "", err
 	}
 
-	id, err := dm.runContainerInPod(pod, container, netNamespace, "")
+	id, err := dm.runContainerInPod(pod, container, netNamespace, "", "")
```

Comment:
There were some further comments about that.  I think the pid mode has to be passed in, IIUC